### DISCUSSION TITLE
error when upgrade flet to 0.25.2

### DISCUSSION THREAD

**Discussion by abdokhalil07** [flet-dev/flet#4583] [Q&A] [answered]
### Question

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
flet-cli 0.25.1 requires flet==0.25.1, but you have flet 0.25.2 which is incompatible.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
How/where do you get that error?

**Reply by abdokhalil07** [reply]
![flet cmd](https://github.com/user-attachments/assets/fe820c60-4ec8-472a-8609-f55bfde0db50)

when update flet in cmd use " pip install flet --upgarde"

**Reply by FeodorFitsner** [reply, marked as answer]
Upgrade it with:

```
pip install 'flet[all]' --upgarde
```
